Cilsil/Program.cs

```diff
@@ -174,7 +174,7 @@ private static IEnumerable<string> GetAssemblies(IEnumerable<string> paths)
                 }
                 catch (FileNotFoundException e)
                 {
-                    Log.WriteError($"Input DLL {e.FileName} not found.");
+                    Log.WriteLine($"Input DLL {e.FileName} not found.");
```

Comment:
You should keep this as write error as it is actually an error. The other cases in which WriteError appears within the parsers are less crucial -- adjust those to write a warning.